domain/src/main/java/org/oppia/domain/util/StateRetriever.kt

```diff
@@ -184,7 +184,7 @@ class StateRetriever @Inject constructor(
           "HasDenominatorEqualTo", "HasNumeratorEqualTo" -> ruleSpecBuilder.putInput(
             inputName,
             InteractionObject.newBuilder()
-              .setReal(inputsJson.getDouble(inputName))
+              .setNonNegativeInt(inputsJson.getInt(inputName))
```

Comment:
You can have negative numerators as well right? Should we have a negative int options as well?

Reply:
i.e. hasnumeratorequalto will set signedInt
and hasDenominatorEqualTo will set nonNegativeInt

Reply:
Ah you’re right, and that matches Oppia web. Will update shortly.

Reply:
Done.